model/components/pinpoint.yml

```diff
@@ -0,0 +1,64 @@
+---
+components:
+
+  # Pinpoint Agent
+  - id: pinpoint-agents
```

Comment:
is it on purpose that the id is in plural form "pinpoint-agents" instead of "pinpoint-agent"?

Reply:
Actually, there isn't any purpose. I've taken a look at what others have done. Few of them that I've looked first was in plural. I'll change them:)